login-workflow/src/new-architecture/i18nInstance/provider.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * @packageDocumentation
+ * @module I18nInstanceProvider
+ */
+
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { i18nBluiInstance } from '.';
+import { I18nInstanceProviderProps } from './types';
+
+export const I18nInstanceProvider: React.FC<React.PropsWithChildren<I18nInstanceProviderProps>> = (props) => {
```

Comment:
Not sure why we have a separate component for this. The `<I18nextProvider>` should just be the first thing returned in the AuthContextProvider...we shouldn't need an extra wrapper component.